index.html

```diff
@@ -0,0 +1,254 @@
+				<div>
+					<p>1 Unit: $2599.00</p>
+				</div>
+				<div>
+					<p>5 Units: $2,299ea</p>
+				</div>
+				<div>
+					<p>10 Units: $1,999.00 ea</p>
+				</div>
+			</section>
+		</article>
+		<article class="card">
+			<header class="headingdiv">
+				<h1 class="title">Mac Mini</h1>
+			</header>
+			<section>
+				<div class="imgdiv macminiimgdiv">
+					<img src="images/macmini.jpeg" alt="Mac Mini">
+				</div>
+				<div class="descriptiondiv">
+					<p>Hardware: 3.0GHz dual-core Intel Core i7 (Turbo Boost up to 3.5GHz) with 4MB on-chip shared L3 cache, 2TB Fusion Drive, 16GB of 1600MHz LPDDR3 memory, Intel Iris Graphics</p>
+					<p>Display: None</p>
+					<p>Software: Pages, Numbers, Keynote, Photos, iMovie, GarageBand, OS X</p>
+					<p>Audio: Audio in port, 3.5 mm headphone jack, IR receiver</p>
+					<p>Ports: Two Thunderbolt 2 ports (up to 20 Gbps), Four USB 3 ports (up to 5 Gbps), HDMI port, SDXC card slot, Gigabit Ethernet port.</p>
+				</div>
+				<div class="availabilitydiv">
+					<p>Available</p>
+				</div>
+			</section>
+			<section class="specificationsection">
+				<h3>Specifications</h3>
+				<div>
+					<p>Height: 1.4 in</p>
+
+					<p>Width: 7.7 in</p>
+				</div>
+				<div>
+					<p>Weight: 2.7 lbs</p>
+
+				</div>
+				<footer>
+					<p>Specifications become invalid June 2017</p>
+				</footer>
+			</section>
+			<section class="pricingsection">
+				<h3>
+					Product Pricing
+				</h3>
+				<div>
+					<p>1 Unit: $1,499.00</p>
+				</div>
+				<div>
+					<p>5 Units: $1,299.00 ea</p>
+				</div>
+				<div>
+					<p>10 Units: $1,099.00 ea</p>
+				</div>
+			</section>
+		</article>
+		<article class="card">
+			<header class="headingdiv">
+				<h1 class="title">Mac Pro</h1>
+			</header>
+			<section>
+				<div class="imgdiv">
+					<img src="images/thunderboltdisplay.jpg" alt="Thunderbolt Display">
+				</div>
+				<div class="descriptiondiv">
+					<p>Hardware: Screen size: 27 inches (diagonal viewable), Screen type: TFT Active-Matrix LCD, Resolution: 2560 x 1440 pixels, Color resolution: 16.7 million, Aspect ratio: 16:9, Brightness: 375 cd/m2, Contrast ratio: 1000:1, Response Time: 12 ms, Built-in FaceTime HD camera with microphone</p>
+					<p>Accessory Kit: User's Guide (English).</p>
+					<p>Ports: Thunderbolt</p>
+				</div>
+				<div class="availabilitydiv">
+					<p>Available</p>
+				</div>
+			</section>
+			<section class="specificationsection">
+				<h3>Specifications</h3>
+				<div>
+					<p>Height: 19.35 in</p>
+
+					<p>Width: 25.7 in</p>
+				</div>
+				<div>
+					<p>Weight: 23.5 lbs</p>
+
+				</div>
+				<footer>
+					<p>Specifications become invalid June 2020</p>
+				</footer>
+			</section>
+			<section class="pricingsection">
+				<h3>
+					Product Pricing
+				</h3>
+				<div>
+					<p>1 Unit: $999.00</p>
+				</div>
+				<div>
+					<p>5 Units: $799.00 ea</p>
+				</div>
+				<div>
+					<p>10 Units: $699.00 ea</p>
+				</div>
+			</section>
+		</article>
+	</div>
+</body>
+</html>
```

Comment:
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior